tests/test_flows_settings.py

```diff
@@ -0,0 +1,71 @@
+import os
+
+# Must import test base before the Controller
+from base import FlowTest, FlowStep
+
+from seedsigner.controller import StopFlowBasedTest
+from seedsigner.models import SettingsDefinition
+from seedsigner.models.settings import Settings
+from seedsigner.models.settings_definition import SettingsConstants
+from seedsigner.gui.screens.screen import RET_CODE__BACK_BUTTON
+from seedsigner.views.view import MainMenuView
+from seedsigner.views import settings_views
+
+
+
+class TestSettingsFlows(FlowTest):
+
+    def test_persistent_settings(self):
+        """ Basic flow from MainMenuView to enable/disable persistent settings """
+        # Which option are we testing?
+        settings_entry = SettingsDefinition.get_settings_entry(SettingsConstants.SETTING__PERSISTENT_SETTINGS)
+
+        # No settings file should exist before we enable persistent settings
+        assert os.path.exists(Settings.SETTINGS_FILENAME) == False
+
+        self.run_sequence([
+            FlowStep(MainMenuView, button_data_selection=MainMenuView.SETTINGS),
+            FlowStep(settings_views.SettingsMenuView, button_data_selection=settings_entry.display_name),
+            FlowStep(settings_views.SettingsEntryUpdateSelectionView, button_data_selection=settings_entry.get_selection_option_display_name_by_value(SettingsConstants.OPTION__ENABLED)),
+            FlowStep(settings_views.SettingsEntryUpdateSelectionView, screen_return_value=RET_CODE__BACK_BUTTON),
+            FlowStep(settings_views.SettingsMenuView, screen_return_value=StopFlowBasedTest()),
```

Comment:
`screen_return_value=StopFlowBasedTest()` is a pattern no longer required.  Also, import not needed on line 6.